crm_integration_xblock/crm_integration_xblock.py

```diff
@@ -34,6 +34,10 @@ class CrmIntegration(StudioEditableXBlockMixin, XBlock):
     and holding the private data we don't want to send to the browsers.
     """
 
+    # Show the XBlock when masquerading as a specific user.
+    # Use with precaution! Don't modify the state of the user.
+    show_in_read_only_mode = True
```

Comment:
Does this change anything else in the way the xblock behaves, or the assumption is that we should go and implement logic of the kind ``if preview_mode: do nothing``

Reply:
I tested it in a local env with the proper SalesForce configurations and it doesn't change the behavior.  Implementing a preview_mode where the user can see the XBlock but can't interact with it when masquerading would depend on the client needs.

Reply:
Good, then this is just a cosmetic change. I still think it is worth it